infrastructure/src/main/java/org/corfudb/infrastructure/CompactorService.java

```diff
@@ -0,0 +1,150 @@
+package org.corfudb.infrastructure;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import lombok.NonNull;
+import lombok.Setter;
+import org.corfudb.runtime.CorfuCompactorManagement.CheckpointingStatus;
+import org.corfudb.runtime.CorfuCompactorManagement.CheckpointingStatus.StatusType;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.DistributedCompactor;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.TxnContext;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.concurrent.SingletonResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static org.corfudb.runtime.view.TableRegistry.CORFU_SYSTEM_NAMESPACE;
+
+/**
+ * Orchestrates distributed compaction
+ * <p>
+ * Created by Sundar Sridharan on 3/2/22.
+ */
+public class CompactorService implements ManagementService {
+
+    @Setter
+    private static Duration livenessTimeout = Duration.ofMinutes(1);
+
+    private final ServerContext serverContext;
+    private final SingletonResource<CorfuRuntime> runtimeSingletonResource;
+
+    private final ScheduledExecutorService orchestratorThread;
+    private final InvokeCheckpointing checkpointerJvmManager;
+
+    private final CompactionTriggerPolicy compactionTriggerPolicy;
+    private CompactorLeaderServices compactorLeaderServices;
+    private CorfuStore corfuStore;
+
+    private final Logger syslog;
+
+    CompactorService(@NonNull ServerContext serverContext,
+                     @NonNull SingletonResource<CorfuRuntime> runtimeSingletonResource,
+                     @NonNull InvokeCheckpointing checkpointerJvmManager,
+                     @NonNull CompactionTriggerPolicy compactionTriggerPolicy) {
+        this.serverContext = serverContext;
+        this.runtimeSingletonResource = runtimeSingletonResource;
+
+        this.orchestratorThread = Executors.newSingleThreadScheduledExecutor(
+                new ThreadFactoryBuilder()
+                        .setNameFormat("Cmpt-" + serverContext.getServerConfig().get("<port>") + "-chkpter")
+                        .build());
+        this.checkpointerJvmManager = checkpointerJvmManager;
+        this.compactionTriggerPolicy = compactionTriggerPolicy;
+        syslog = LoggerFactory.getLogger("syslog");
```

Comment:
I don't think we should be adding such a generic logger. You can see how other classes can name their loggers.